install

```diff
@@ -77,6 +77,15 @@ def macos_version
   @macos_version ||= Version.new(`/usr/bin/sw_vers -productVersion`.chomp[/10\.\d+/])
 end
 
+def should_install_command_line_tools
```

Comment:
Typical Ruby style would have this method be called `should_install_command_line_tools?`